src/oca/go/src/goca/schemas/host/host.go

```diff
@@ -73,10 +73,18 @@ type Share struct {
 }
 
 type Datastores struct {
-	DiskUsage int `xml:"DISK_USAGE,omitempty"`
-	FreeDisk  int `xml:"FREE_DISK,omitempty"`
-	MaxDisk   int `xml:"MAX_DISK,omitempty"`
-	UsedDisk  int `xml:"USED_DISK,omitempty"`
+	DiskUsage int       `xml:"DISK_USAGE,omitempty"`
+	FreeDisk  int       `xml:"FREE_DISK,omitempty"`
+	MaxDisk   int       `xml:"MAX_DISK,omitempty"`
+	UsedDisk  int       `xml:"USED_DISK,omitempty"`
+	Datastore Datastore `xml:"DS,omitempty"`
+}
+
+type Datastore struct {
+	ID      int `xml:"ID,omitempty"`
+	TotalMB int `xml:"TOTAL_MB,omitempty"`
```

Comment:
Same as in https://github.com/OpenNebula/one/blob/08bb8f7684a2b773f33aac3bd3e81e8e2f23c50b/src/oca/go/src/goca/schemas/datastore/datastore.go#L49-L51